src/cljs/rems/administration/administration.cljs

```diff
@@ -63,7 +66,10 @@
                                                         :rems.administration/license
                                                         :rems.administration/create-license}
                                                       selected)
-                                       :active)]]])
+                                       :active)]]
+   [to-blacklist [:nav-item :nav-link (when (contains? #{:rems.administration/blacklist}
```

Comment:
I think there's some possibility for refactoring to remove duplication and tedium.

Reply:
done